src/internal/modules/account/forms/set-permissions.js

```diff
@@ -65,8 +67,10 @@ const form = (request, permission) => {
     }
   }));
 
-  const newAccountEmail = get(request, 'yar._store.newInternalUserAccountEmail');
-  f.fields.push(fields.hidden('newUserEmail', {}, newAccountEmail));
+  if (newUser) {
+    const newAccountEmail = get(request, 'yar._store.newInternalUserAccountEmail');
```

Comment:
The _ often denotes a variable that should be considered private.  Should we be using the documented API, e.g.:
```
request.yar.get('newInternalUserAccountEmail');
```